<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbValidationDialog</class>
 <widget class="QDialog" name="DbValidationDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>645</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="pBtnShowDetails">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Details</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableOverview">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableDetails">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>240</y>
     <width>1200</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
